libcontainer/configs/config.go

```diff
@@ -212,6 +212,9 @@ type Config struct {
 	// RootlessCgroups is set when unlikely to have the full access to cgroups.
 	// When RootlessCgroups is set, cgroups errors are ignored.
 	RootlessCgroups bool `json:"rootless_cgroups,omitempty"`
+
+	// TimeOffsets specifies the offset for supporting time namespaces.
+	TimeOffsets map[string]specs.LinuxTimeOffset `json:"timeOffsets,omitempty"`
```

Comment:
To match the rest of the state json:

```suggestion
	TimeOffsets map[string]specs.LinuxTimeOffset `json:"time_offsets,omitempty"`
```